docs/air-pool/create-airpool-substrate-asset.md

```diff
@@ -0,0 +1,44 @@
+---
+sidebar_position: 1
+---
+
+# Creating Substrate Native / AssetHub Asset AirPool
+
+This article focuses on how to create your first Substrate Native / AssetHub Asset reward pool on AirLyft.
```

Comment:
Explain what are substrate native tokens and asset hub assets. Give links to polkadot documentation as well